instruments/newport/newportesp301.py

```diff
@@ -1154,10 +1153,7 @@ def stop_motion(self):
         """
         Stop all motion on axis. With programmed deceleration rate
         """
-        try:
-            self._newport_cmd("ST", target=self.axis_id)
-        except NewportError as e:
-            raise NewportError(e)
+        self._newport_cmd("ST", target=self.axis_id)
```

Comment:
I don't understand what this `try` statement tried to do. If an error already occured, why would it then raise an error, the error should have already been raised? That would make this redudant.

Reply:
I agree. I'd have to look into the git logs to see if we previously had something else in there. Either way, you are correct